saleor/userprofile/i18n.py

```diff
@@ -37,6 +50,22 @@
     'zip': pgettext_lazy('Address field', 'ZIP code')}
 
 
+class CustomPhonePrefix(PhonePrefixSelect):
+
+    def __init__(self, initial=None):
+        choices = phone_prefixes
+        if initial:
+            self.initial = phone_prefixes[initial]
+
```

Comment:
Empty blank lines in simple functions aren't necessary, it doesn't add much to readability.